qemu/tests/scsi_mq_check.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+
+
+def get_mq_from_guest(session, params, cmd, pattern):
+    """
+    Get multi queue number from guest.
+    """
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        logging.error("cmd running failed with output %s" % output)
+    if params.get("os_type") == 'linux':
+        queues = len(re.findall(pattern, output, re.I | re.M))
+    if params.get("os_type") == "windows":
+        queues = re.search(pattern, output, re.I | re.M).group(1)
+    if queues:
+        return queues
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check device multi-queue function enabled.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = float(params.get("login_timeout", 240))
+    driver_name = params.get("driver_name")
+    session = vm.wait_for_login(timeout=timeout)
+    num_queues = params.get("num_queues")
+    pattern = params.get("pattern")
+    check_queue_cmd = params.get("check_queue_cmd")
+
+    if params.get("os_type") == "windows" and driver_name:
+        session = utils_test.qemu.windrv_check_running_verifier(session, vm,
+                                                                test, driver_name,
+                                                                timeout)
+    error_context.context("Get the num_queues info", logging.info)
+    check_queue_cmd = utils_misc.set_winutils_letter(session, check_queue_cmd)
```

Comment:
'utils_misc.set_winutils_letter' should be operated in above "windows" code block.

Reply:
has updated, thanks comment~